ocamldoc/odoc_ast.ml

```diff
@@ -1137,6 +1139,7 @@ module Analyser =
               else
                 None
             in
+            let comment_opt = Odoc_sig.analyze_alerts comment_opt val_desc.Parsetree.pval_attributes in
```

Comment:
That's clever and probably the best way to source attributes information from the AST while keeping the separated extraction of documentation comments.